site/src/components/WorkspaceScheduleButton/WorkspaceScheduleButton.tsx

```diff
@@ -88,7 +84,7 @@ export const WorkspaceScheduleButton: React.FC<WorkspaceScheduleButtonProps> = (
               disabled={!deadlinePlusEnabled()}
               onClick={onDeadlinePlus}
             >
-              <Tooltip title={Language.editDeadlinePlus}>
+              <Tooltip title={t("WorkspaceScheduleButton.editDeadlinePlus")}>
```

Comment:
I have not had a chance to run this yet but does the capitalized `W` cause any issues?  The rest are lowercased.

Reply:
haha, yes, that will cause issues. Good catch!